gcp/vars.tf

```diff
@@ -0,0 +1,50 @@
+variable "region" {
+  default = "us-west1"
+}
+
+variable "zone" {
+  description = "The zone where the cluster will be deployed [a,b,...]"
+  default     = "a"
+}
+
+variable "subnet" {
+  description = "The name of the existing subnet where the machines will be deployed"
+}
+
+variable "nodes" {
+  description = "The number of nodes to deploy."
+  type        = number
+  default     = "1"
+}
+
+variable "disks" {
+  description = "The number of local disks on each machine."
+  type        = number
+  default     = 1
+}
+
+variable "image" {
+  # See https://cloud.google.com/compute/docs/images#os-compute-support
+  # for an updated list.
+  default = "ubuntu-os-cloud/ubuntu-1804-lts"
```

Comment:
Any reason not to use 20.04?

Reply:
I used what was in our internal modules, so I hadn't noticed. But I will test and update it.